CHANGES_NEXT_RELEASE

```diff
@@ -1,2 +1,4 @@
+- Add: apikey to findType in memory registry for groups
```

Comment:
Maybe better (from user perpective rather than developer point of view?)

```suggestion
- Add: include apikey in queries to memory registry for groups
```